calendar/page.js

```diff
@@ -604,9 +659,18 @@ class ColTypesFetcher {
       this._colTypesPromise = ColTypesFetcher.getTypes(this._tableId, this._colIds);
     }
   }
-  async getColTypes() {
+
+  async wait() {
```

Comment:
is this used anywhere?

Reply:
Good catch, used to be, but now it's not.